test/parallel/test-debugger-heap-profiler.js

```diff
@@ -23,11 +24,11 @@ const filename = tmpdir.resolve('node.heapsnapshot');
       await cli.waitForInitialBreak();
       await cli.waitForPrompt();
       await cli.command('takeHeapSnapshot()');
-      JSON.parse(readFileSync(filename, 'utf8'));
+      assert.ok(fs.existsSync(filename));
```

Comment:
Does this change the test? The original test would've thrown an error if the file wasn't valid JSON?

Reply:
other tests are already covers that. i don't think this is needed.

Reply:
This test is used as a regression test for a write race condition so checking for valid JSON might be a good idea. See https://github.com/Trott/io.js/commit/c24deca89be5908539bed6e80747266343d76cf1.

Reply:
I see. It appears that this test timeouts pretty often. Any recommendation on how to avoid?

Reply:
You could still reduce the operation count by not reading the file to a string. JSON.parse accepts a buffer as input.